spec/unit/documents/request/shipment_order_spec.rb

```diff
@@ -110,6 +110,23 @@ module Request
         assert_equal "DEF", order_details[1]['ItemNumber']
       end
 
+      it 'merges the back to reality parts into one bundle' do
```

Comment:
looks like this description was leftover from a prior spec, but this spec is for black tie set right?